server/athenian/api/models/state/models.py

```diff
@@ -640,3 +640,19 @@ class DashboardChartGroupBy(create_time_mixin(created_at=True, updated_at=True),
     DashboardChartGroupBy.jira_issue_types,
     DashboardChartGroupBy.jira_labels,
 )
+
+
+class InstallationProgress(create_time_mixin(created_at=False, updated_at=False), Base):
+    """Tracking installation progress for github accounts"""
+
+    __tablename__ = "installation_progress"
+
+    github_account_id = Column(Integer(), primary_key=True)
+    account_created = Column(TIMESTAMP(timezone=True))
+    fetch_started = Column(TIMESTAMP(timezone=True))
+    fetch_completed = Column(TIMESTAMP(timezone=True))
+    consistency_started = Column(TIMESTAMP(timezone=True))
+    consistency_completed = Column(TIMESTAMP(timezone=True))
+    precompute_started = Column(TIMESTAMP(timezone=True))
+    precompute_completed = Column(TIMESTAMP(timezone=True))
+    current_status = Column(Text())
```

Comment:
Let's delete this *for now*. I need us to create something absolutely minimal and ASAP. If we make the status, we will have to think what statuses can we have, when to update and what, consider different edge cases, etc.

Reply:
noone is forcing us to use all the columns of the table from the begining.
we can create a table "as is" and add the features to the pipeline and API when we will be ready

Reply:
Let's migrate to additional fields in the future. To avoid seduction.

Reply:
Sorry, but how one text field is making it harder? We already have a PR to MD that uses it.

Reply:
addition - PR to cloud-common is ready too. and if we cut the columns it will slow us to implement changes instead of getting some features faster.
and if we delete precompute timestamps from this table we will use all the rest immediately after releasing all the features that in process now

Reply:
Yes, I'd still like to have that field for the bot.

Agree about precomputer, we will need a second table then?

Reply:
precompute- fields dropped. account_created field dropped. others (including status) is used by actual written logic in metadata. the point is that status can have a voluntary (apart of some statuses that would be fixed) value and this can be used for some specific messages like "paused", "delayed" etc.

Reply:
@dennwc we have a field "precomputed bool" in the table "repository_sets"
so it would be right to add a couple new fields to it. and provide a logic to fill this fields from the precomputer directly (not with event handler).

Reply:
It makes sense to write to metadata :+1: